src/plugins/charts/public/services/colors/mapped_colors.test.ts

```diff
@@ -61,7 +61,7 @@ describe('Mapped Colors', () => {
     mappedColors.mapKeys(arr);
 
     const colorValues = _(mappedColors.mapping).values();
-    expect(colorValues.includes(seedColors[0])).toBe(false);
+    expect(colorValues).not.toContain(seedColors[0]);
```

Comment:
:information_source: A left over from debugging the tests, since this produces way better output, but I'd just leave it with this clearer API.